jupyter/pytorch+llmcompressor/ubi9-python-3.12/pyproject.toml

```diff
@@ -91,7 +91,7 @@ explicit = true
 # https://github.com/astral-sh/uv/issues/4422
 override-dependencies = [
     # vllm is a requirement, however, as compressed-tensors is pinned, this can't be installed together
-    "compressed-tensors",
+    "compressed-tensors==0.12.2",
```

Comment:
Cam we remove the override-dependencies completely?

Reply:
Done, pylock looks still good